nl/seg6_linux.go

```diff
@@ -0,0 +1,121 @@
+package nl
+
+import (
+	"errors"
+	"fmt"
+	"net"
+)
+
+type Ipv6SrHdr struct {
```

Comment:
Minor: I think `IP` should be capitalized `IPv6SrHdr`

Reply:
OK. Will change to `IPv6SrHdr`.